tests/integration/mutation/create/field_kinds/field_kind_json_test.go

```diff
@@ -66,24 +141,31 @@ func TestMutationCreate_WithJSONFieldGivenInvalidJSON_Error(t *testing.T) {
 			},
 			testUtils.Request{
 				Request: `mutation {
-					create_Users(input: {name: "John", custom: "{\"tree\": \"maple, \"age\": 250}"}) {
+					create_Users(input: {name: "John", custom: "hello"}) {
 						_docID
 						name
 						custom
 					}
 				}`,
-				ExpectedError: `Argument "input" has invalid value {name: "John", custom: "{\"tree\": \"maple, \"age\": 250}"}.
```

Comment:
todo: Please adjust the test name to match the new test. `TestMutationCreate_WithJSONFieldGivenSimpleString_Error` has the same problem

Reply:
done